source/vulkan/runtime_vk.cpp

```diff
@@ -898,8 +942,10 @@ bool reshade::vulkan::runtime_vk::compile_effect(effect_data &effect)
 			image_binding.imageView = _backbuffer_texture_view[info.srgb];
 			break;
 		case texture_reference::depth_buffer:
-			// TODO (set to backbuffer for now to make validation layers happy)
-			image_binding.imageView = _backbuffer_texture_view[0];
+			if(_depthstencil_image_view == VK_NULL_HANDLE)
+				image_binding.imageView = _default_depthstencil_view;
```

Comment:
Maybe add a comment here as to why this is set (because it technically is incorrect and is just done to make validation layers happy).

Reply:
OK, will see what happens when i comment those lines...

Reply:
Done.